code/modules/mob/living/simple_animal/abnormality/waw/rose_sign.dm

```diff
@@ -401,19 +418,52 @@
 /mob/living/simple_animal/hostile/rose_summoned/CanAttack(atom/the_target)
 	return FALSE
 
-/mob/living/simple_animal/hostile/rose_summoned/death()
+/mob/living/simple_animal/hostile/rose_summoned/Destroy()
 	if(!killed || !status_target)
 		return ..()
 	if(flower_damage_type && master)
 		master.summoned_roses -= src
 		master.ChangeResistance(flower_damage_type, (master.damage_coeff.getCoeff(flower_damage_type) + 0.3), update = TRUE)
 	if(status_target.has_status_effect(/datum/status_effect/stacking/crownthorns))
 		status_target.remove_status_effect(STATUS_EFFECT_THORNS)
-	density = FALSE
-	animate(src, alpha = 0, time = 10 SECONDS)
-	QDEL_IN(src, 10 SECONDS)
 	..()
```

Comment:
Since you changed this from death() to Destroy(), remember that it HAS to end with `return ..()`
```suggestion
	return ..()
```